reference/usage-billing.mdx

```diff
@@ -41,25 +41,14 @@ To purchase compute credits:
 
 ### Spending limit
 
-Setting a monthly spending limit allows you to control usage-related costs.
-
-The spending limit applies to the following usage-related costs:
-- Data loading
-- Query compute
-- Storage
-
-The spending limit doesn’t apply to the following costs:
-- Your pricing plan
-- Add-ons
-
-This means that even if you reach your spending limit, add-ons and actions that don’t generate usage costs continue working as usual.
-
-The spending limit only takes into account the usage-related costs you’re billed for at the end of your billing period. For the purposes of determining your spending limit, pre-purchased credits don’t count.
+Setting a monthly spending limit allows you to control costs.
 
 <Warning>
-If your organization reaches the spend limit in a billing period, Axiom immediately prevents actions that generate further usage costs. For example, you can’t run queries, you can’t ingest more data, and your monitors can’t query your data. This can cause interruptions to your service.
+The spending limit takes into account the costs you’re billed for at the end of your billing period. If your organization reaches the spend limit in a billing period, Axiom immediately prevents actions that generate further usage costs. For example, you can’t run queries, you can’t ingest more data, and your monitors can’t query your data. This can cause interruptions to your service.
```

Comment:
>>For example, you can’t run queries, you can’t ingest more data, and your monitors can’t query your data.
I think we may need to be more clear here that you can't run queries if you have used up what is included in your plan, etc etc same for ingest

Reply:
For example once you have consumed the query hours included with your platform fee and you are in breach of your spend limits threshold you will not be able to query, run monitors or view dashboards.